vllm/model_executor/layers/pooler.py

```diff
@@ -23,12 +24,13 @@ class Pooler(nn.Module):
     3. Returns structured results as `PoolerOutput`.
 
     Attributes:
-        pooling_type: The type of pooling to use (LAST, AVERAGE, MAX).
+        pooling_type: The type of pooling to use (LAST, MAX, CLS).
```

Comment:
```suggestion
        pooling_type: The type of pooling to use (LAST, ALL, CLS).
```

Right now, we don't have `MAX`...